docs/dev_tools/third_party.md

```diff
@@ -36,6 +36,14 @@ For example, to export to uVision5 with the K64F target run:
 A `*.uvproj` file is created in the root folder of the project. 
 You can open this project file with uVision5.
 
+When exporting from mbed CLI, you will get a project that compiles with the debug profile. More information on the debug profile may be found in the [build profiles documentation](build_profiles.md#debug-profile). For example, this means that compiling within UVision 5 after this export:
+
```

Comment:
I recommend changing the first two sentences to the following for active voice: 

"When you export from mbed CLI, you create a project that compiles with the `debug` profile. You can find more information on the `debug` profile in the [build profiles documentation](build_profiles.md#debug-profile)."